console/src/app/modules/org-table/org-table.component.ts

```diff
@@ -65,6 +70,10 @@ export class OrgTableComponent {
     this.requestOrgsObservable$.pipe(switchMap((req) => this.loadOrgs(req))).subscribe((orgs) => {
       this.dataSource = new MatTableDataSource<Org.AsObject>(orgs);
     });
+
+    this.mgmtService.getIAM().then((iam) => {
+      this.globalOrgId = iam.globalOrgId;
```

Comment:
please use the `defaultOrgId` attribute (globalOrgId is a leftover from v1 and deprecated)
and also name it like that in this file

Reply:
done